PullRequestClosed/PullRequestClosed.test.ts

```diff
@@ -0,0 +1,150 @@
+    github.context.payload.pull_request.merged = true;
+    github.context.payload.pull_request.base = { ref: 'master' };
+    await runAction('Title', "renovate[bot]");
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Loading PR #42",
+      "Invoked jira.moveIssue('KEY-1234', 'Resolve issue', {\"resolution\":{\"id\":\"10000\"}})",
+      "Done",
+    ]);
+  });
+
+  it('is-eng-xp-squad Bot PR unmerged PR cancels issue', async () => {
+    process.env['INPUT_IS-ENG-XP-SQUAD'] = 'true';
+    await runAction('Title', "renovate[bot]");
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Loading PR #42",
+      "Invoked jira.moveIssue('KEY-1234', 'Resolve issue', {\"resolution\":{\"id\":\"10001\"}})",
+      "Done",
+    ]);
+  });
+
+  it('Unmerged PR for ticket created by automation closed', async () => {
+    await runAction('KEY-5678 Title');
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Invoked jira.moveIssue('KEY-5678', 'Cancel Issue', {\"resolution\":{\"id\":\"10001\"}})",
+      "Done"
+    ]);
+  });
+
+  it('Unmerged PR for pre-existing issue closed', async () => {
+    await runAction('KEY-1234 Title');
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Skipping issue cancellation for creator Creator of KEY-1234",
+      "Done"
+    ]);
+  });
+
+  it('Jira issue does not exist', async () => {
+    const logSpy = jest.spyOn(core, 'setFailed').mockImplementation(() => { });
+    try {
+      await runAction('FAKE-1234 Title');
+      expect(logSpy).toHaveBeenCalledWith('Scaffolding did not expect FAKE-1234');
+    } finally {
+      logSpy.mockRestore();
+    }
+  });
+
+  it('PR merged into release branch', async () => {
+    const releaseBranches = ['master', 'main', 'branch-0.0.0'];
+    for (const branchName of releaseBranches) {
+      github.context.payload.pull_request.merged = true;
+      github.context.payload.pull_request.base = { ref: branchName };
+      await runAction('KEY-1234 Title');
+      expect(logTester.logsParams).toStrictEqual([
+        "Loading PR #42",
+        `Invoked jira.transitionIssue('KEY-1234', 'transition: Merge into master 10000', null`,
+        "Done",
+      ]);
+      logTester.logsParams = [];
+    }
+  });
+
+  it('PR merged into non-release branch', async () => {
+    github.context.payload.pull_request.merged = true;
+    github.context.payload.pull_request.base = { ref: 'user/branch' };
+    await runAction('KEY-5678 Title');
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Invoked jira.transitionIssue('KEY-5678', 'transition: Merge into branch 10001', null",
+      "Done",
+    ]);
+  });
+
+  it('Merge Pull Request - transision is missing', async () => {
+    github.context.payload.pull_request.merged = true;
+    github.context.payload.pull_request.base = { ref: 'user/branch' };
```

Comment:
This shouldn't matter, let's use the usual suspect
```suggestion
    github.context.payload.pull_request.base = { ref: 'master' };
```